db/counting_index.go

```diff
@@ -100,13 +100,6 @@ func GetCountingIndex(kv KVStore, name []byte) (CountingIndex, error) {
 	if errors.Cause(err) == ErrNotExist || total == nil {
 		return nil, errors.Wrapf(err, "counting index 0x%x doesn't exist", name)
 	}
-	if kvFillPercent, ok := kv.(KVStoreWithBucketFillPercent); ok {
```

Comment:
GetCountingIndex() only returns the index for read, so no need to set the fill percent, which is only for write operation